ci-operator/step-registry/idp/htpasswd/idp-htpasswd-commands.sh

```diff
@@ -105,21 +105,39 @@ function set_users () {
     oc get $TARGET_RESOURCE -o json > "${oauth_file_src}"
     jq $IDP_FIELD' += [{"htpasswd":{"fileData":{"name":"cucushift-htpass-secret"}},"challenge":"true","login":"true","mappingMethod":"claim","name":"cucushift-htpasswd-provider","type":"HTPasswd"}]' "${oauth_file_src}" > "${oauth_file_dst}"
     oc replace -f "${oauth_file_dst}"
+   
 
-    # wait for oauth-openshift to rollout
-    wait_auth=true
-    expected_replicas=$(oc get deployment oauth-openshift -n "$OAUTH_NAMESPACE" -o jsonpath='{.spec.replicas}')
-    while $wait_auth;
+    echo "Wait htpasswd ready up to 10 minutes"
+    auth_ready=false
+    count=0
+    while [[ $count -lt 40 ]]
     do
+        expected_replicas=$(oc get deployment oauth-openshift -n "$OAUTH_NAMESPACE" -o jsonpath='{.spec.replicas}')
```

Comment:
This line should be put before the loop because the replicas won't change.

Reply:
Fixed